src/components/scs/ScSheet/ScSheet.js

```diff
@@ -236,7 +236,47 @@ const ScSheet = ({ sc, scWithPr, classes, intl }) => {
   );
 
   const handleSubmit = () => {
-    // TODO: submitting data and sending to backend
+    const mapToDTO = field => {
+      return {
+        title: field.title,
+        evaluation: typeof field.evaluation === 'number' ? field.evaluation : 1,
+        percentage: field.percentage,
+        description: field.description,
+        achievement: field.achievement,
+        weight: field.weight,
+        comment: field.comment
+      };
+    };
+
+    const data = {
+      dailyBusiness: dailyBusinessFields.map(mapToDTO),
+      project: projectFields.map(mapToDTO),
+      workEfficiency: mapToDTO(workEfficiencyFields),
+      workQuality: mapToDTO(workQualityFields),
+      skillsInTheFields: mapToDTO(skillsInTheFieldsFields),
+      impactOnTeam: mapToDTO(impactOnTeamFields),
+      serviceQuality: mapToDTO(serviceQualityFields),
+      impactOnCompany: mapToDTO(impactOnCompanyFields),
+      skillsWeightPercentage: prCategoriesWeightPercentage
+    };
+
+    savePerformanceData(
+      sc.id,
+      user.isReviewerInSc(sc) ? 'reviewer' : 'employee',
+      data,
+      info,
+      error
+    ).then(() => {
+      if (user.isOwnerInSc(sc)) {
+        if (!sc.statusSet.includes(SC_STATUS.EMPLOYEE_SUBMITTED)) {
+          addStatus(sc.id, SC_STATUS.EMPLOYEE_SUBMITTED, error);
+        }
+      } else if (user.isReviewerInSc(sc)) {
+        if (!sc.statusSet.includes(SC_STATUS.REVIEWER_SUBMITTED)) {
+          addStatus(sc.id, SC_STATUS.REVIEWER_SUBMITTED, error);
+        }
+      }
+    });
```

Comment:
Shouldn't it fetch new data about sc after submit? Otherwise, user has to refresh the page to see updated SC.